CHANGELOG.md

```diff
@@ -37,6 +37,12 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
 * We now use Doxygen version 1.8.17 to build our documentation
   ([\#1500](https://github.com/seqan/seqan3/pull/1500)).
 
+#### Search
+
+* The `seqan3::fm_index_cursor::extend_right()`, `seqan3::bi_fm_index_cursor::extend_right()` and
```

Comment:
This is still under `New features` but should be under `notable bug fixes` :)

Reply:
Thx, did I place the test in the right place?